eoscostarica/lifebank
# feat(backend): add send email action

Resolves #573 

### What does this PR do?
Enable the send email again option if user have not received email verification yet

### How should this be tested?
- Make run
- Login without having verified the email
- Check email is send
